app/presenters/vendor_api/application_presenter/interviews.rb

```diff
@@ -8,6 +8,6 @@ def schema
   end
 
   def interviews
-    application_choice.interviews.order(updated_at: :desc)
+    application_choice.interviews.sort(&:updated_at).reverse
```

Comment:
Also this query ... I don't know if I should replace it with pure ruby to avoid the N+1 query. I'll roll back for now but I am happy to discuss.